client/logic/ui/map_editor_ui.gd

```diff
@@ -6,6 +6,7 @@ signal unit_selected(unit_id: String, unit_scene: PackedScene)
 signal tile_selected(atlas_id: Vector2i)
 signal edit_selected
 signal remove_selected
+@warning_ignore("unused_signal")
```

Comment:
I think this signal can be dropped is from my old code, and I don't think in this code will work fine

Reply:
It is used in the [map resize menu script](https://github.com/Codename-Recon/Gridlock/blob/5c11c9c1677b50886fed238eb2b1916812a4bfde/client/logic/ui/map_editor_resize_menu.gd#L12)